ofl/notosansduployan/METADATA.pb

```diff
@@ -10,20 +10,26 @@ fonts {
   filename: "NotoSansDuployan-Regular.ttf"
   post_script_name: "NotoSansDuployan-Regular"
   full_name: "Noto Sans Duployan Regular"
-  copyright: "Copyright 2018 Google LLC. All Rights Reserved."
+  copyright: "Copyright 2018-2019 David Corbett. Copyright 2019-2022 Google LLC. All Rights Reserved."
+}
+fonts {
+  name: "Noto Sans Duployan"
+  style: "normal"
+  weight: 700
+  filename: "NotoSansDuployan-Bold.ttf"
+  post_script_name: "NotoSansDuployan-Bold"
+  full_name: "Noto Sans Duployan Bold"
+  copyright: "Copyright 2018-2019 David Corbett. Copyright 2019-2022 Google LLC. All Rights Reserved."
 }
-subsets: "menu"
 subsets: "duployan"
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+source {
+  repository_url: "https://github.com/notofonts/duployan.git"
+  archive_url: "https://github.com/notofonts/duployan/releases/download/NotoSansDuployan-v3.001/NotoSansDuployan-v3.001.zip"
+}
 is_noto: true
 languages: "chn_Dupl"  # Chinook Jargon, Duployan
 languages: "de_Dupl"  # German, Duployan
 languages: "fr_Dupl"  # French, Duployan
-sample_text {
-  masthead_full: "𛰳𛱰𛱘𛱢"
-  masthead_partial: "𛰠𛰏"
-  styles: "𛱔𛱕𛱖𛱗𛱘𛱙𛱚 𛱩𛱪𛱰𛱱𛱲𛱳𛱴 𛰀𛰁𛰂𛰃𛰄𛰅𛰆 𛲔𛲕𛲖𛲗𛲘𛲙𛲜 𛰎𛰏𛰐𛰑𛰒𛰓𛰔 𛱢𛱣𛱤𛱥𛱦𛱧𛱨"
-  tester: "𛰀𛰁𛰂𛰃𛰄𛰅𛰆 𛰱𛰲𛰳𛰴𛰵𛰶𛰷 𛱼𛲀𛲁𛲂𛲃𛲄𛲅 𛰎𛰏𛰐𛰑𛰒𛰓𛰔 𛱆𛱇𛱈𛱉𛱊𛱋𛱌 𛱢𛱣𛱤𛱥𛱦𛱧𛱨 𛱔𛱕𛱖𛱗𛱘𛱙𛱚 𛲔𛲕𛲖𛲗𛲘𛲙𛲜 𛱩𛱪𛱰𛱱𛱲𛱳𛱴 𛰜𛰝𛰞𛰟𛰠𛰡𛰢"
-  poster_sm: "𛰡𛰓𛰵𛱖𛰐𛱗𛱴𛰶𛱙𛰆"
-  poster_md: "𛰷𛰓𛰆𛲜𛲗 𛰶𛱗𛱖"
-  poster_lg: "𛰞𛰆"
-}
```

Comment:
Is that okay for the sample text to get removed?
also missing primary script

Reply:
Many of these sample strings are unrealistic and look messy and unreadable in the new font, so they should at least be modified if not removed.

Reply:
They should not be removed as we don't have any sample strings in the lang repo, but we should put some in there in the future and remove from here.